frozen.py

```diff
@@ -1,54 +1,65 @@
 import gym
 import numpy as np
+import argparse
 
 env = gym.make('FrozenLake-v0')
 env.reset()
 
-NUM_ACTIONS=env.action_space.n
-NUM_STATES= env.observation_space.n
-q_table=np.zeros((NUM_STATES, NUM_ACTIONS))
+NUM_ACTIONS = env.action_space.n
+NUM_STATES = env.observation_space.n
 """print q_table"""
 
-
 # Q-Learning Parameters
 
-G=.99
-L=.75
-
```

Comment:
removing unnecessary white space and descriptive variable names makes the code more readable.